src/pretix/static/pretixpresale/js/ui/main.js

```diff
@@ -82,7 +82,13 @@ var form_handlers = function (el) {
                     Math.abs(+new Date(opts.minDate) - new Date()) < Math.abs(+new Date(opts.maxDate) - new Date())
             ) ? opts.minDate : opts.maxDate;
         }
-        $(this).datetimepicker(opts);
+        $(this).datetimepicker(opts).on("dp.hide", function() {
+            var $dtp = $(this).data("DateTimePicker");
+            var currentViewDate = $dtp.viewDate();
+            window.setTimeout(function () {
+                $dtp.viewDate(currentViewDate);
+            }, 500);
+        });
         if ($(this).parent().is('.splitdatetimerow')) {
```

Comment:
We also have support for data-min and data-max in pretixcontrol and we also render QuestionForm there, so I suggest to make the same change in pretixcontrol/ui/main.js

Reply:
Good point! I added the fix (with a comment why this fix is needed) to pretixcontrol/ui/main.js as well.